### DISCUSSION TITLE
How to set filename_as_title?

### DISCUSSION THREAD

**Discussion by becks0815** [stashapp/stash#3720] [Q&A] [answered]
Hi,
I have found the hint that there is an option to set the filename as title in the config file, but neither I have found any entry in the stash config.yml nor in the wiki. Where do I have to place this option?

**Comment by DogmaDragon** [comment, marked as answer]
Hi,

since v0.17 the filename is used by default if no title is set. There is also a plugin to set the title to be the same as filename https://docs.stashapp.cc/add-ons/plugins/#titlefromfilename
